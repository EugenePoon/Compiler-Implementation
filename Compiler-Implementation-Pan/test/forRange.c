int main()
{
    int i;
    for i in (3..9) {
        print i;
    }
}