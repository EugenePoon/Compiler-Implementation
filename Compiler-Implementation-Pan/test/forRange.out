24 19 0 5 25 15 1 13 0 3 12 15 -1 16 29 13 11 22 15 -1 13 13 11 0 1 1 12 15 -1 13 11 0 9 7 18 15 21 0